From adf0563b1b54a5c8e9433867e300b72c73438272 Mon Sep 17 00:00:00 2001
From: Ranjan Dutta <ranjan.dutta@intel.com>
Date: Tue, 6 Sep 2022 07:43:33 +0800
Subject: [PATCH 51/60] Revert "drm/gem: Properly annotate WW context on
 drm_gem_lock_reservations() error"

This reverts commit 1ff71d4f532bbc53d0986b65310ff0e31f3c73b3.
---
 drivers/gpu/drm/drm_gem.c | 4 ++--
 1 file changed, 2 insertions(+), 2 deletions(-)

diff --git a/drivers/gpu/drm/drm_gem.c b/drivers/gpu/drm/drm_gem.c
index 5979af230eda0..69c2c079d8036 100644
--- a/drivers/gpu/drm/drm_gem.c
+++ b/drivers/gpu/drm/drm_gem.c
@@ -1277,7 +1277,7 @@ drm_gem_lock_reservations(struct drm_gem_object **objs, int count,
 		ret = dma_resv_lock_slow_interruptible(obj->resv,
 								 acquire_ctx);
 		if (ret) {
-			ww_acquire_fini(acquire_ctx);
+			ww_acquire_done(acquire_ctx);
 			return ret;
 		}
 	}
@@ -1302,7 +1302,7 @@ drm_gem_lock_reservations(struct drm_gem_object **objs, int count,
 				goto retry;
 			}
 
-			ww_acquire_fini(acquire_ctx);
+			ww_acquire_done(acquire_ctx);
 			return ret;
 		}
 	}
-- 
2.25.1

